### ISSUE TITLE
Performance issues on transactions paging

### ISSUE THREAD

**Issue by ales-tsurko** [Moneyfactory-io/blockscout-frontend#38]
Reported by testers.
<img width="940" alt="Screenshot 2024-04-29 at 18 53 46" src="https://github.com/Atleta-network/blockscout/assets/8686568/2d8ce72a-e42f-46fa-b833-0d8e34609672">

**Comment by alwayscolorblind** [comment]
Yes, here is actually a little delay (< 0.3ms) between click and the beginning of some changes. 
Need to fix? Looks like react render issue, may be hard to debug and fix. Eth blockscout is also has a little delay.
